ui/app/AppLayouts/Wallet/controls/ManageTokensCommunityTag.qml

```diff
@@ -47,16 +53,59 @@ Control {
             Layout.preferredHeight: visible ? 16 : 0
             asset.width: 16
             asset.height: 16
-            asset.isImage: true
-            visible: !!asset.source
+            visible: root.full && !!asset.source
+            asset.name: !!root.communityImage ? root.communityImage : "help"
+            asset.isImage: !!root.communityImage
         }
-        StatusBaseText {
+
+        RowLayout {
+            spacing: 2
             Layout.fillWidth: true
-            font.pixelSize: Style.current.tertiaryTextFontSize
-            font.weight: Font.Medium
-            text: root.text
-            elide: Text.ElideRight
-            color: enabled ? Theme.palette.directColor1 : Theme.palette.baseColor1
+
+            StatusBaseText {
+                font.pixelSize: Style.current.tertiaryTextFontSize
+                font.weight: Font.Medium
+                text:  {
+                    if (d.unknown) {
+                        if (communityNameToolTip.visible) {
+                            if (!root.full) {
+                                return StatusQUtils.Utils.elideAndFormatWalletAddress(root.communityName)
+                            }
+                            return qsTr("Community %1").arg(StatusQUtils.Utils.elideAndFormatWalletAddress(root.communityName))
+                        }
+                        if (!root.full) {
```

Comment:
I don't get this use case. Could you provide more details?

Reply:
Text change: unknown community vs unknown

Reply:
Yeah, but which is the scenario to show the "unknown"?